main.php

```diff
@@ -117,6 +117,8 @@
 require_once('core/startup/ErrorControlChain.php');
 require_once('core/startup/ParameterConfirmationToken.php');
 
+ini_set('display_errors', 0);
```

Comment:
Could this be done in such a way that it's only suppressed AFTER logging is setup, and some kind of check to prove that it's able to display errors? I'm scared of white screen of death appearing. Probably some time just after set_error_handler?